packages/databricks-vscode/package.json

```
@@ -203,6 +203,13 @@
                 "enablement": "databricks.context.activated",
                 "category": "Databricks"
             },
+            {
+                "command": "databricks.bundle.selectActiveProjectFolder",
+                "icon": "$(gear)",
+                "title": "Select Databricks Project Folder",
```

```suggestion
                "title": "Select a Databricks project folder",
```
I think we want to stick to sentence casing for directives.